### PR TITLE
[TRITON] Add Softmax Triton Kernel

### PR DESCRIPTION
**Pull request by lucas-santos-amd**

### CODE REVIEW

File: aiter/ops/triton/softmax.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import torch
+import triton
+import triton.language as tl
+
+
+@triton.jit
+def softmax_kernel_online(
```
Comment (rahulbatra85) [inline aiter/ops/triton/softmax.py]: please add _ in front of the name.
Thanks!

**Review by rahulbatra85 — APPROVED** [review]

**Review by rahulbatra85 — APPROVED** [review]

**Review by rahulbatra85 — APPROVED** [review]

**Review by vgokhale — APPROVED** [review]
